homeassistant/components/media_player/universal.py

```diff
@@ -328,6 +332,11 @@ def current_source(self):
         return self._child_attr(ATTR_INPUT_SOURCE)
 
     @property
+    def source_list(self):
+        """List of available input sources."""
+        return self._source_list
```

Comment:
That's an option, but why pick MQTT, and not make it generic (like with the universal media player)? Maybe somebody wants to create a command line media player? I could just use a config like this, right?

```yaml
  commands:
    turn_on:
      service: mqtt.publish
      data:
        topic: "cec/on"
        payload: "5"
```